cypress/support/cypress-commands/commands.js

```diff
@@ -35,15 +35,25 @@ Cypress.Commands.add(
   (key, callType = CONSTANTS.SUPPORTED_CALLTYPES.FIREBOLTCALLS) => {
     // Reading the data from combinedJson based on key.
     let fireboltData;
+    let logMessage = CONSTANTS.NO_DATA_FOR_THE_KEY + key;
     if (callType == CONSTANTS.SUPPORTED_CALLTYPES.FIREBOLTMOCKS) {
       fireboltData = UTILS.getEnvVariable(CONSTANTS.COMBINEDFIREBOLTMOCKS)[key];
     } else if (callType == CONSTANTS.SUPPORTED_CALLTYPES.SET_RESPONSE_JSON) {
       fireboltData = UTILS.getEnvVariable('setResponseJson')[key];
     } else {
-      fireboltData = UTILS.getEnvVariable(CONSTANTS.COMBINEDFIREBOLTCALLS)[key];
+      if (key) {
```

Comment:
I think this logic should be apart of the glue command. We don't want to set this runtime variable for non setter/getter calls.

Reply:
removed this changes from here and added in glue itself. will move it to some function in optimization ticket